doc/examples/jasmine/karma.conf.js

```diff
@@ -0,0 +1,51 @@
+// Karma configuration
+module.exports = function(config) {
+	config.set({
+		// base path that will be used to resolve all patterns (eg. files, exclude)
```

Comment:
Can we remove the comments here? IMO they're only helpful for people setting up Karma (you), not for people reading this code (us).